test/github.test.js

```diff
@@ -123,6 +124,66 @@ describe('github', function() {
     });
   });
 
+  describe('fetch_current_reviewers()', function() {
+    const stub = sinon.stub();
+    const octokit = {
+      pulls: {
+        listRequestedReviewers: stub,
+      },
+    };
+
+    beforeEach(function() {
+      github.getOctokit.returns(octokit);
+    });
+
+    it('fetch current reviewers - user only', async function() {
```

Comment:
```suggestion
    it('fetches current reviewers - user only', async function() {
```

minor nit: shouldn't this be "it fetches" rather than "it fetch"? Same for the other additions